docs/integrations/generic-commands-reputation.md

```diff
@@ -0,0 +1,43 @@
+---
+id: generic-commands-reputation
+title: Generic Reputation Commands
+---
+
+
+## Background and motivation
+
+XSOAR has an abudance of integrations with reputation providers for example VirusTotal, AlienVault OTX, MISP etc. 
+Every Integration that returns a reputation about an indicator must implement the generic repuation commands and calculate [DBot Score](../dbot)
+
+## Generic Reputation Commands needs to be supported
+
+### **file**
+
+**Description:** Runs reputation on a File.
```

Comment:
We should mention the argument name as well, and that is mandatory and default, and that the command should return the `File` context standard with a link to the schema. Maybe add the YML of the command so they can easily copy and paste.

Reply:
A YML here will be a copy paste of the context standards. maybe i can link to it instead.. WDYT?